cmd/create_test.go

```diff
@@ -21,6 +21,7 @@ kind: Project
 metadata:
   name: Test
 spec:
+  public: true
```

Comment:
Any chance that we can make this into `visibility: public/private`?

Reply:
Sure

Reply:
Awsm.